libs/matdbg/src/ShaderInfo.cpp

```diff
@@ -94,12 +94,12 @@ bool getMetalShaderInfo(const ChunkContainer& container, ShaderInfo* info) {
     return true;
 }
 
-bool getGlShaderInfo(const ChunkContainer& container, ShaderInfo* info) {
-    if (!container.hasChunk(ChunkType::MaterialGlsl)) {
+bool getGlShaderInfo(const ChunkContainer& container, ShaderInfo* info, ChunkType chunkType) {
```

Comment:
Just roughly looking the three methods: `getGlShaderInfo` `getMetalShaderInfo`, `getVkShaderInfo`, looks like they are all the same except for the chunktype?  Maybe consolidate?

Reply:
Done.